recipes/qt.py/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "Qt.py" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "209fd4909241f08983e9cdf8768341b7f56adcc37cc4abacd0bca6ad8620479f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+    - pyside2
+  imports:
+    - Qt
+
+about:
+  home: http://github.com/mottosso/Qt.py
+  license: MIT
+  license_family: MIT
```

Comment:
Can you add `license_file: LICENSE` here?

Reply:
Ok, done. Is the actual `LICENSE` file going to be generated?

Reply:
Adding this line would copy the file with the name `LICENSE` from the source folder to the conda package.

Reply:
See https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
Looks like I need to add a `LICENSE` file. Is that really required for the MIT license?

Should I add a `LICENSE` next to `meta.yaml` and this should work?

Reply:
Ok, thanks!